src/managing-workflow/security-considerations.md

```diff
@@ -41,15 +41,23 @@ that hosts applications intended for a limited audience-- e.g. applications for
 an organization. You can enforce cluster-wide IP whitelisting by enabling whitelists, then
 attaching an annotation to the router:
 
-$ kubectl --namespace=deis annotate replicationcontroller deis-router router.deis.io/nginx.enforceWhitelists=true
-$ kubectl --namespace=deis annotate replicationcontroller deis-router router.deis.io/nginx.defaultWhitelist="0.0.0.0/0"
+    $ kubectl --namespace=deis annotate deployments/deis-router router.deis.io/nginx.enforceWhitelists=true
+    $ kubectl --namespace=deis annotate deployments/deis-router router.deis.io/nginx.defaultWhitelist="0.0.0.0/0"
 
 The format is the same for the controller whitelist but you need to specify the whitelist directly
 to the controller's service. For example:
 
-$ kubectl --namespace=deis annotate service deis-controller router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+    $ kubectl --namespace=deis annotate service deis-controller router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
 
 And the same applies to applications. For example, to apply a whitelist to an application named
 `example`:
 
-$ kubectl --namespace=example annotate service example-web router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+    $ kubectl --namespace=example annotate service example-web router.deis.io/whitelist="10.0.1.0/24,121.212.121.212"
+
+Application level whitelist can also be done using deis client. To add/remove/list addresses of an application whitelist, use `deis whitelist`:
```

Comment:
> Application level whitelist can also be done using deis client.

Is a little awkward. Maybe "Application level **whitelists** can also be done using **the** deis client."

There also are not any example of listing application whitelists.